odoo/exceptions.py

```diff
@@ -48,8 +48,8 @@ class RedirectWarning(Exception):
     :param str button_text: text to put on the button that will trigger
         the redirection.
     """
-    def __init__(self, message, action, button_text):
-        super().__init__(message, action, button_text)
+    def __init__(self, message, action, button_text, additionnal_context=None):
```

Comment:
It's a detail, but adds additional_context in docstring and removes an `n` from additionnal